spec/features/events_feature_spec.rb

```diff
@@ -1,8 +1,10 @@
 require "rails_helper"
+require "timecop"
 
 feature "Events" do
   before do
-    create(:event, name: "Katas and hang")
+    starts_at = Time.local(2018, 9, 1, 10, 0, 0)
```

Comment:
It doesn't really matter for this test, but as a general practice it's usually best to use `Time.utc` rather than `Time.local` in testing. UTC is less brittle -- if the test was looking for a certain date string on the page (rather than freezing time to the date) then the test would be flaky because it would fail during the hours when local time is a different date than UTC time (which also might confusingly vary for different developers based on their own time zone).